lib/uvobot.rb

```diff
@@ -8,19 +8,19 @@ def run(release_date)
     if @scraper.issue_ready?(release_date)
       notify_announcements(release_date)
     else
-      @notifier.new_issue_not_published
+      @notifiers.each { |n| n.send(:new_issue_not_published) }
     end
   end
 
   private
 
   def notify_announcements(release_date)
-    page_info, announcements = @scraper.get_announcements(release_date)
+    page_info, announcements = @scraper.get_full_announcements(release_date)
```

Comment:
Toto je zbytocne pridavanie roboty. `DiscourseNotifier` by mal zabezpecit aj to stahovanie detailov. Tak to vieme aj pripadne hodit do async queue a sme v pohode. Totu ked padne jedno stahovanie, tak sme hotovi.